README.md

```diff
@@ -1,6 +1,21 @@
-# similar-npm-packages
+<div align="center">
+  <a href="https://bundlephobia.com/">
+    <img src="logo.png" width="278" height="168" alt="similar-npm-packages" />
+  </a>
+</div>
 
-Similar packages suggestions for the NPM ecosystem
+<div align="center">
+  Similar packages suggestions for the NPM ecosystem.<br />
+  <a href="https://bundlephobia.com/">Bundlephobia</a> team project
+</div>
+
+## How to add a package or category
+
+The best way to help us improve our registry is to create a Pull Request updating `src/similar-npm-packages.ts` file.
```

Comment:
Currently folks can just submit an issue, and I do a monthly round-up of tickets and add those as suggestions. Creating a PR is appreciated, but optional.

One of the reasons for this is that the additions of `tags` is a manual process that cannot be easily automated (I've had to manually tweak the weights of tags every time a new category is created).

Do you think folks that don't have time to raise a PR should be able to submit issues?